proto/evm/v1beta1/query.proto

```diff
@@ -84,3 +84,10 @@ message QueryCommandResponse {
   string key_id = 4 [ (gogoproto.customname) = "KeyID" ];
   uint32 max_gas_cost = 5;
 }
+
+message BurnerInfoRequest {
+  string chain = 1;
+  string address = 2;
```

Comment:
Why is `address` not of type `types.Address` but `string`?

Reply:
Because `types.Address` is a byte array, and my intention is to pass the arguments in the form of `http://blabla/evm/v1beta/burner_info/chain/address`. The way grpc query works it does not expect us to implement a rest handler like the legacy querier.

Reply:
also, I just tried to change this field into a `types.Address` and I get a compilation error on the `service.pb.gw.go` file. It looks like the resulting code does not even attempt to cast the the byte slice into `types.Address`, it simples attempts to assign the value directly:

```
message BurnerInfoRequest {
  string chain = 1;
  bytes address = 2 [ (gogoproto.nullable) = false, (gogoproto.customtype) = "Address" ];
}
```

```
	val, ok = pathParams["address"]
	if !ok {
		return nil, metadata, status.Errorf(codes.InvalidArgument, "missing parameter %s", "address")
	}

	protoReq.Address, err = runtime.Bytes(val)
```

Reply:
changed query endpoint to receive a `types.Address`, though from the testing i did, it looks like we will need to pass a base64 of its bytes. Not sure how straightforward this is for the ms team.

Reply:
That seems inconvenient to use when debugging via CLI. Can we keep it as string input?

Reply:
Fair point. @sammy1991106 @haiyizxx thoughts?

Reply:
gRPC itself isn't user-friendly at all with CLI since the gRPC client has to load the protobuf to begin with. I don't think CLI users are more important here but other systems who call us in the context of gRPC. In such a case, developers may be wondering about casing if it was a string, which I'd like to avoid any confusion with.

Reply:
leaving the protobuf as is then